### ISSUE TITLE
WCAG 2.2 Violation - There are instances of list items not marked up as lists within the interface.

### ISSUE THREAD

**Issue by ivrouwe-ubc** [ubc/wp-comments-chat-ui#25]
### Issue

There are some elements that are presented as lists that are not marked up as lists.

#### This violates the following WCAG (Web Content Accessibility Guidelines) 2.2 Success Criteria:

- [1.3.1 Info and Relationships](https://www.w3.org/TR/WCAG22/#info-and-relationships)

### Instances

- #27 
- #28 

### User Impact

Some users (such as screen reader users) will be unaware that there is a list present. Users who need navigate to (or through) lists using assistive technologies will be unable to do so for these instances.

#### Further information:

1. ["H48: Using ol, ul and dl for lists or groups of links | WAI | W3C"](https://www.w3.org/WAI/WCAG22/Techniques/html/H48)
2. [The "Intent" section of "Understanding Success Criterion 1.3.1: Info and Relationships | WAI | W3C"](https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships.html#intent)

#### Using your browser's developer tools:

1. Inspect the element in question using your browser's developer tools.
2. Notice that one or more of the following conditions is not true:
    1. The parent element that houses the list items is not marked up as a `<ul>` or `<ol>` element.
    2. The list items are not marked up as `<li>` elements.
    3. The list items are not direct children of the list element.
    4. (If the element is using the `role` attribute to designate lists or list items [instead of `<ul>`/`<ol>`, and `<li>`]):
        1. The list does not have `role="list"` set on it.
        2. The list items do not have `role="listitem"` set on them.

#### Using a screen reader (such as VoiceOver, NVDA, JAWS, or TalkBack):

1. Activate the screen reader.
2. Navigate to the element (or pull up a list of headings on the page), and notice that the element is not announced (or included) as a heading.

### How To Fix The Issue

Ensure that the element and its children are marked up as a list (`<ul>` or `<ol>` element) and list items (`<li>` elements), respectively.

#### WCAG 2.2 Conformant Code Example

```
<!-- A generic example of an unordered list -->
<ul>
	<li>Item A</li>

	<li>Item B</li>
	[…]
</ul>

<!-- A generic example of an ordered list -->
<ol>
	<li>Item A</li>

	<li>Item B</li>
	[…]
</ol>
```
